lib/Intercom.js

```diff
@@ -57,6 +59,11 @@ export default class Intercom {
     if (!window.intercomSettings) { window.intercomSettings = {} }
     window.intercomSettings.app_id = this.appId
 
+    Object.keys(this.config).forEach((k) => {
+      const newKey = k.replace(/([A-Z])/g, '_$1').toLowerCase()
+      window.intercomSettings[newKey] = this.config[k]
+    })
+
```

Comment:
Could you replace `k` with `key` and `newKey` with `snakeCaseKey` to make it more verbose please?

```javascript
      const snakeCaseKey = key.replace(/([A-Z])/g, '_$1').toLowerCase()
```

Reply:
sure no problem. i noticed also i hadn't updated the README.md to include config in the settings but I was thinking this may be generated from defaults.js which i did update. Does the README also need to be updated?

Reply:
Ah yeah it's not generated, if you could update that too that'd be brill :)